src/BaGet.UI/src/DisplayPackage/DisplayPackage.tsx

```diff
@@ -313,6 +314,8 @@ class DisplayPackage extends React.Component<IDisplayPackageProps, IDisplayPacka
 
               <p>{(!this.state.package.authors) ? 'Unknown' : this.state.package.authors}</p>
             </div>
+
+            <RepoDetails url={this.state.package.projectUrl}/>
```

Comment:
The `ICatalogEntry` I made is incorrect. The `projectUrl` property should be nullable. See: https://github.com/loic-sharma/BaGet/blob/f0ee2c57c221fd6f0e866f051eb5892e464b85f0/src/BaGet.UI/src/DisplayPackage/Registration.tsx#L26

Reply:
Will do!